### PR TITLE
Review feature/hero

### PR DESCRIPTION
**Pull request by elawrobel**
Hey, 

in this PR I pushed new hero. Would be grate if you have time to review it.

E.

### CODE REVIEW

File: app/containers/NewApp/index.js [hunk 1]
```diff
@@ -35,16 +35,10 @@ export default function App() {
...
       </Helmet>
       <Menu />
-      <Title />
-      <Header />
-      <Switch>
-        <Route exact path="/" component={HomePage} />
-        <Route path="/events" component={() => <span>events</span>} />
-        <Route path="" component={NotFoundPage} />
-      </Switch>
+      <Hero />
```
Comment (ucanthurtme) [inline app/containers/NewApp/index.js:41]: I'm wondering is it displayed correctly ? I mean you've removed all of the routing part - that's why I'm asking. If you are not sure, don't worry let me know then I'm gonna check it.

File: app/components/Menu/index.js [hunk 2]
```diff
@@ -15,13 +15,19 @@ const Wrapper = styled.div`
...
-  justify-content: space-around;
+  justify-content: space-between;
   align-items: center;
+  padding: 1rem 4rem;
 `;
 
 const Nav = styled.ul`
   display: flex;
   flex-direction: row;
+
```
Comment (ucanthurtme) [inline app/components/Menu/index.js:26]: noob question: shouldn't we try to at least implement mobile first solution, is it still valid solution within styled components ?

File: app/components/Hero/index.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+const Wrapper = styled.section`
+  padding: 4em;
+  background: ${props => props.theme.colors.white};
+  border-bottom: 1px solid black;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  @media (max-width: 960px) {
```
Comment (ucanthurtme) [inline app/components/Hero/index.js:15]: mobile first ?

File: app/containers/NewApp/index.js (hunk 1 above)
Comment (elawrobel) [reply app/containers/NewApp/index.js:41]: Right... bad decision.

Comment (ucanthurtme) [reply app/containers/NewApp/index.js:40]: Are we ready to approve & merge it ?

**Pull request merged**